src/GraphQL.Tests/Introspection/SchemaIntrospectionTests.cs

```diff
@@ -48,41 +48,15 @@ public async Task validate_core_schema_pascal_case(IDocumentWriter documentWrite
 
             var json = await documentWriter.WriteToStringAsync(executionResult);
 
-            ShouldBe(json, "IntrospectionResult".ReadJsonResult());
-        }
-
-        [Theory]
-        [ClassData(typeof(DocumentWritersTestData))]
-        public async Task validate_core_schema_doesnt_use_nameconverter(IDocumentWriter documentWriter)
```

Comment:
After adding schema validation and fixing tests, it is no longer possible for name converter not to be called. Query must be defined and must have at least one field.